src/openrct2/ride/TrackDesign.cpp

```diff
@@ -855,14 +855,12 @@ static void track_design_add_selection_tile(int16_t x, int16_t y)
     gMapSelectionTiles.push_back(CoordsXY(x, y));
 }
 
-static void track_design_update_max_min_coordinates(int16_t x, int16_t y, int16_t z)
+static void track_design_update_max_min_coordinates(const CoordsXYZ coords)
```

Comment:
Have you made sure to check if this function is always properly called with big Z coordinates?

Reply:
Big Z coordinates?

Reply:
I've checked and this one is fine.

Reply:
I'll mark this as resolved then.

@frutiemax this PR explains what are big and small coords: https://github.com/OpenRCT2/OpenRCT2/pull/12192